lib/ansible/plugins/action/service.py

```diff
@@ -31,6 +31,10 @@ class ActionModule(ActionBase):
         'systemd': ['pattern', 'runlevel', 'sleep', 'arguments', 'args'],
     }
 
+    # HACK: list of unqualified service manager names that are/were built-in, we'll prefix these with `ansible.legacy` to
+    # avoid collisions with collections search
+    BUILTIN_SVC_MGR_MODULES = set(['openwrt_init', 'service', 'systemd', 'sysvinit'])
```

Comment:
Do we need `openwrt_init` in this list? I'm unclear what happens if we try to run `ansible.legacy.openwrt_init` since we don't have that in `ansible-base` anymore. It gets picked up from `community.general` if that is installed, but maybe that is the goal.

Reply:
Yep, we're relying on the redirection to Do The Right Thing- this list is based on what was in 2.9